packages/plugin-node-uncaught-exception/test/uncaught-exception.test.js

```diff
@@ -0,0 +1,59 @@
+const { describe, it, expect } = global
+
+const Client = require('@bugsnag/core/client')
+const schema = require('@bugsnag/core/config').schema
+const VALID_NOTIFIER = { name: 't', version: '0', url: 'http://' }
+const plugin = require('../')
+
+describe('plugin: node uncaught exception handler', () => {
+  it('should listen to the process#uncaughtException event', () => {
+    const before = process.listeners('uncaughtException').length
+    const c = new Client(VALID_NOTIFIER)
+    c.setOptions({ apiKey: 'api_key' })
+    c.configure()
+    c.use(plugin)
+    const after = process.listeners('uncaughtException').length
+    expect(before < after).toBe(true)
```

Comment:
we should verify that this has increased by 1, unless this adds multiple listeners?